src/include/api/ivf_pq_index.h

```diff
@@ -562,22 +588,23 @@ class IndexIVFPQ {
   tiledb_datatype_t id_datatype_{TILEDB_ANY};
   tiledb_datatype_t partitioning_index_datatype_{TILEDB_ANY};
   std::unique_ptr<index_base> index_;
+  DistanceMetric distance_metric;
```

Comment:
```suggestion
  DistanceMetric distance_metric_;
```
and also I think we need to define a default here, currently we'll have undefined behavior if the user doesn't pass it in the config. Would be good to add tests checking that this is set correctly a) when it's not explicitly set b) when it's explicitly set.